src/OrchardCore.Modules/OrchardCore.Https/Startup.cs

```diff
@@ -34,6 +46,13 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSingleton(new RewriteOptions());
             
             services.TryAddEnumerable(ServiceDescriptor.Transient<IConfigureOptions<RewriteOptions>, RewriteOptionsHttpsConfiguration>());
+
+            services.AddHsts(options =>
+            {
+                options.Preload = true;
```

Comment:
Preloading can have a permanent impact on users and should be an explicit choice. See here for additional explanation: https://github.com/roots/trellis/issues/727

You should make this false by default and add this as a comment in the code so that it doesn't get changed in the future.